civildatetime.md

```diff
@@ -0,0 +1,225 @@
+
+  *	If the year value modulo 4 != 0, the range is 1 to 28, inclusive.
+
+  *	If the year value modulo 400 == 0, the range is 1 to 29, inclusive.
+
+  * If the year value modulo 100 == and year value % 400 != 0, the range is 1 to 28, inclusive.
+
+  * If the year value modulo 4 == 0 and year value % 100 != 0, the range is 1 to 29, inclusive.
+
+#### Hour Number
+The Hour Number is a positive integer corresponding to a particular hour.  It is called the hour value and is identified by an integer in the range of 0 to 23, inclusive.  It must have a Number value and may not be NaN.
+
+#### Minute Number
+The Minute Number is a positive integer corresponding to a particular minute.  It is called the minute value and is identified by an integer in the range of 0 to 59, inclusive.  It must have a Number value and may not be NaN.
+
+#### Second Number
+The Second Number is a positive integer corresponding to a particular second.  It is called the second value and is identified by an integer in the range of 0 to 59, inclusive.  It must have a Number value and may not be NaN.
+
+#### Millisecond Number
+The Millisecond Number is a positive integer corresponding to a particular millisecond.  It is called the millisecond value and is identified by an integer in the range of 0 to 999, inclusive.  It must have a Number value and may not be NaN.
+
+#### Nanosecond Number
+The Nanosecond Number is a positive integer corresponding to a particular nanosecond.  It is called the nanosecond value and is identified by an integer in the range of 0 to 999999, inclusive.  It must have a Number value and may not be NaN.
+
+#### From(date, time)
+The abstract operation From takes a CivilDate Object and a CivilTime Object returns a new CivilDateTime object with the corresponding argument values for year, month, day, hour, minute, second, millisecond, and nanosecond.
+  
+#### Plus(Object)
+The abstract operation Plus returns a new CivilDateTime object that may have 0 to 8 values copied from the CivilDateTime object on which the operation was performed.  The new CivilDateTime object will have one or more updated values that depend on terminal values of an Object passed in as the argument.  The argument Object consists of a range of 1 to 8 (inclusive) key/value pairs.  Each pair consists of a nonterminal symbol and a corresponding terminal Number.  An argument Object may have one and only one of each of the nonterminal symbols "Year", "Month", "Day", "Hour", "Minute", "Second", "Millisecond", and "Nanosecond".
+
+#### FromDate(date, zone)
+Something?
+
+## The CivilDateTime Constructor
+The CivilDateTime constructor is the %CivilDateTime% intrinsic object.  When called as a constructor, it creates and initializes a new CivilDateTime object.
+The CivilDateTime constructor is a single function.
+The CivilDateTime constructor is subclassable.
+The length property of the CivilDateTime constructor function is a range of 2 to 5, inclusive.
```

Comment:
Should this be `a range of 5 to 8, inclusive`?

Reply:
Thank you.  Fixed.